README.md

```
@@ -67,14 +67,19 @@ npm run unified-dev
 ## Deployment
 
-### Render Deployment
+```
+npm run build
+npm run start
+```
+
+deployed on digital ocean using pm2
 
-This application is designed to be deployed on Render. Use the following settings:
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)